### ISSUE TITLE
Validation programatically disabled

### ISSUE THREAD

**Issue by georgemoralis** [rrohm/fx-validation#5]
I have a case in which validation might be different if a checkbox is selected on a form.

It should be able to disable validation for a specific component on demand

**Comment by rrohm** [comment]
You are right, I can imagine use cases like this. I think, this might be a property like the "enable"/"disable" property. I flag this as enhancement.
